Managers/FrameMan.h

```diff
@@ -547,6 +540,9 @@ namespace RTE {
 		static constexpr unsigned short m_BPP = 32; //!< Color depth (bits per pixel).
 
 		int m_GfxDriver; //!< The graphics driver that will be used for rendering.
+		bool m_ForceVirtualFullScreenGfxDriver; //!< Whether to use the borderless window driver. Overrides any other windowed drivers. The driver that will be used is GFX_DIRECTX_WIN_BORDERLESS.
+		bool m_ForceOverlayedWindowGfxDriver; //!< Whether to use the non-overlapped windowed driver. The driver that will be used is GFX_DIRECTX_OVL.
```

Comment:
I actually don't even know what they're supposed to do. Overlayed isn't even supported on my system.
I think i'm gonna go ahead an yeet both of them, and instead reintroduce the dedicated fullscreen driver here so it can be forced on.

Reply:
Sure, I have no issue with that I guess, whatever works best. I guess you can leave the comment as-is if it's all gonna be going away in future 🤷